code/modules/mob/living/carbon/resist.dm

```diff
@@ -3,6 +3,7 @@
 	set category = "IC"
 
 	if(!stat && can_click())
+		setClickCooldown(20)
```

Comment:
`20` is definitely overboard. Use `DEFAULT_QUICK_COOLDOWN` (4) or at least `DEFAULT_ATTACK_COOLDOWN` (8).

Reply:
This was already tried and tested in #4113 and #4138 not long ago, are you sure you want that?